op-node/withdrawals/utils.go

```diff
@@ -48,12 +48,12 @@ func WaitForOutputRootPublished(ctx context.Context, client *ethclient.Client, l
 // WaitForFinalizationPeriod waits until the L1 chain has progressed far enough that l1ProvingBlockNum has completed
 // the finalization period.
 // This functions polls and can block for a very long time if used on mainnet.
-func WaitForFinalizationPeriod(ctx context.Context, client *ethclient.Client, portalAddr common.Address, l1ProvingBlockNum *big.Int) error {
+func WaitForFinalizationPeriod(ctx context.Context, client *ethclient.Client, l2OutputOracleAddr common.Address, l1ProvingBlockNum *big.Int) error {
```

Comment:
Missed one of the callers here: the indexer has a test that breaks.
```
err = withdrawals.WaitForFinalizationPeriod(e2eutils.TimeoutCtx(t, 30*time.Second), l1Client, predeploys.DevOptimismPortalAddr, proveReceipt.BlockNumber)
```
https://github.com/ethereum-optimism/optimism/blob/a06151bd1bd7fade752c618129e4712b8ddf2905/indexer/integration_tests/bedrock_test.go#L211

I would recommend moving the output-oracle address argument to the last argument position of the function, so the method signature has a breaking type change (address->big int for third argument), and any other users of this function we missed (including external users) will notice it as compiler error, rather than silently passing the wrong address.

Reply:
Very good suggestion, thank you!

Reply:
Great call here